RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -19,22 +19,19 @@
     )
 
 _particle_flow_zero_suppression_ECAL_2017 = cms.PSet(
-    thresholds = cms.vdouble(_pfZeroSuppressionThresholds_EB_2017 + _pfZeroSuppressionThresholds_EEminus_2017 + _pfZeroSuppressionThresholds_EEplus_2017
+    thresholds = cms.vdouble(pfZeroSuppressionThresholds_EB + pfZeroSuppressionThresholds_EEminus + pfZeroSuppressionThresholds_EEplus
```

Comment:
After a more careful look, I'd like to propose to move up the change to the arrays in the beginning of the file.

So, please leave this line unchanged and change the arrays above to be
```
#These are expected to be adjusted soon, while the thresholds for older setups will remain unchanged.
_pfZeroSuppressionThresholds_EB_2017 = pfZeroSuppressionThresholds_EB
_pfZeroSuppressionThresholds_EEminus_2017 = pfZeroSuppressionThresholds_EEminus
```

This implies that my current understanding is correct that the adjustments will apply only to 2017+ data periods while the range up to 2016 will remain unchanged.
If this is correct, when we need to make a change, that change will affect only the 2017 arrays above without a requirement to recode this line as well.

Reply:
For sake of a re-reco of all run2 collected data, also 2016 ones should have these thresholds included.

If I understand correctly how these modifiers are handled, with the current configuration if we run on 2016 we have:
particle_flow_zero_suppression_ECAL
while if we run on 2017/2018 the one to be used is:
_particle_flow_zero_suppression_ECAL_2017

Is this correct?

In the future we may need to put different thresholds depending on the year (definitely moving to Tags in DB would help a lot, but the exact values will have to be studied by POGs before deciding on the best values).

Reply:
> For sake of a re-reco of all run2 collected data, also 2016 ones should have these thresholds included.

do you mean the future to-be-tuned thresholds?

How deep does this "for sake of re-reco" argument apply? If common thresholds are expected, they have to be implemented as common explicitly with a single common setting instead of copy-pasting.
Imagine we need to rereco Run-1 data. Would the updated thresholds apply there as well?
If so, then all the modifiers should be removed and we should just use the only thresholds in place in the arrays pfZeroSuppressionThresholds_EB, pfZeroSuppressionThresholds_EEminus, and pfZeroSuppressionThresholds_EEplus.
This file can then be cleaned up by removing everything after line 6

Reply:
Ok, now I see your point.
New commit with the suggested changes just done